packages/runtime/src/lib/constants.ts

```diff
@@ -32,6 +32,8 @@ export const ALGORAND_MAX_TX_ACCOUNTS_LEN = 4;
 // the assets and application arrays combined and totaled with the accounts array can not exceed 8
 export const ALGORAND_MAX_TX_ARRAY_LEN = 8;
 export const MAX_INNER_TRANSACTIONS = 16;
+export const ALGORAND_MAX_LOGS_COUNT = 32;
+export const ALGORAND_MAX_LOGS_LENGTH = 1024;
```

Comment:
let's check if this is defined in JS-SDK as well, if so, we should use SDK directly.

Reply:
unfortunately, this is only exported from `go-algorand`